webui/react/src/types.ts

```diff
@@ -52,6 +52,12 @@ export enum ResourceState { // This is almost CommandState
   Terminated = 'TERMINATED',
 }
 
+// export enum SlotHighState {
```

Comment:
I consdered adding this type to use in Badge component but I went with a custom badge type.

TODO remove

Reply:
I think introducing this type is a better approach for the sake of avoiding having to define colors within JS (basically avoiding CSS-in-JS situation)

Reply:
I added this